css/footer.css

```
@@ -0,0 +1,12 @@
+nav {
+  background-color: white;
+  display: grid;
+  grid-template-rows: max-content;
+  grid-template-columns: 25% 25% 25% 25%;
```

I would suggest to use fractions instead of percentages here. Fractions take only the free space into account (have a look at this [article](https://stackoverflow.com/questions/45090726/the-difference-between-percentage-and-fr-units)). You probably also do not need `grid-template-rows: max-content;`

```suggestion
  grid-template-columns: 1fr 1fr 1fr 1fr;
```
or a bit shorter:
```suggestion
  grid-template-columns: repeat(4, 1fr);
```

---

done